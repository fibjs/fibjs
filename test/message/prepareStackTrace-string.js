'use strict';

function prepareStackTrace(error, stack) {
  return 'custom Error String!';
}
Error.prepareStackTrace = prepareStackTrace;

throw new Error();
