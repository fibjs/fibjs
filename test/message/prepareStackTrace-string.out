*test*message*prepareStackTrace-string.js:*
throw new Error();
^
custom Error String!
